frontend/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
first off... what does this do? mind walking me through its purpose?

Reply:
This is a new thing from yarn. It's apparently a way of linking old yarn with new yarn.
[.yarnrc.yml](https://yarnpkg.com/configuration/yarnrc#nodeLinker)